Add Unit tests for Util Classes

siddharth2010 opened the pull request.

mhutti1 commented on app/src/test/java/org/kiwix/kiwixmobile/utils/NetworkUtilsTest.java:
```diff
@@ -0,0 +1,168 @@
+    //one network is connected
+    when(networkInfo1.getState()).thenReturn(NetworkInfo.State.CONNECTED);
+    when(networkInfo2.getState()).thenReturn(NetworkInfo.State.DISCONNECTING);
+    assertEquals("",true,NetworkUtils.isNetworkAvailable(context));
```
There are a couple places where there are no spaces after commas.

siddharth2010 replied:
@mhutti1 , I have fixed this and updated the branch.

siddharth2010 replied:
@mhutti1 , the Travis build is failing and I have no idea why. It's working perfectly on my local machine.

siddharth2010 replied:
@mhutti1 , The build passed on its own, after restarting it right now. So I think its all good.

siddharth2010 commented:
@mhutti1 , I have fixed this and updated the branch, but the Travis build is failing and I have no idea why. It's working perfectly on my local machine.
